ginivision/src/main/java/net/gini/android/vision/internal/fileimport/providerchooser/ProvidersAppItemViewHolder.java

```diff
@@ -0,0 +1,22 @@
+package net.gini.android.vision.internal.fileimport.providerchooser;
+
+import android.support.annotation.NonNull;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import net.gini.android.vision.R;
+
+class ProvidersAppItemViewHolder extends ProvidersItemViewHolder {
```

Comment:
I prefer my viewholder as an inner class of the adapter, since they are only used there. However it is just personal preferences I guess.

Reply:
Yeah, makes sense, although I'm not a big fan of inner classes I agree that in this case inner classes are the way to go. 👍